sdk/core/corehttp/corehttp/runtime/policies/_universal.py

```diff
@@ -98,6 +98,20 @@ def on_request(self, request: PipelineRequest[HTTPRequestType]) -> None:
         if additional_headers:
             request.http_request.headers.update(additional_headers)
 
+    def on_response(  # pylint: disable=unused-argument
```

Comment:
Nope, this is not ok. Why do we need that?

Reply:
OOB discussion: Did this to avoid mypy errors like ` Cannot instantiate abstract class "HeadersPolicy" with abstract attribute "on_response"`. Since we can't do this, going with using hasattr for both HTTPPolicy and SansIOHTTPPolicy.